sdk.nix

```diff
@@ -113,7 +113,7 @@ rec {
       diffutils dnsmasq dstat
       e2fsprogs elasticsearch
       file findutils
-      gawk gcc glibc glibcLocales gnugrep gnumake gnupg gnused gnutar go gzip
```

Comment:
You still have to keep this bit, that attrset lists all of the packages that are propagated downstream